app/models/claims/state_machine.rb

```diff
@@ -249,7 +250,7 @@ def set_amount_assessed_zero!
     end
 
     def has_been_previously_assessed?
-      claim_state_transitions.map(&:to).any? { |state| %w[authorised part_authorised].include?(state) }
+      claim_state_transitions.map(&:to).any? { |state| PREVIOUSLY_ASSESSED_STATES.include?(state) }
```

Comment:
i think the name makes sense in the context of the method. However the method name is also ambiguos as from a business perspective there are other states that would be considered "previously assessed" (e.g. rejected/refused) but the only possible name i can think of would be `previously_authorised_states` (and ideally a method name change to match).